src/colorizer/CanvasOverlay.ts

```diff
@@ -1,38 +1,94 @@
+import { Vector2 } from "three";
 import { numberToSciNotation } from "./utils/math_utils";
 
-export type ScaleBarOptions = {
-  minWidthPx: number;
+export type StyleOptions = {
   fontSizePx: number;
   fontFamily: string;
   fontColor: string;
+  fontStyle: string;
+};
```

Comment:
Styling options were identical across the scale bar and timestamp, so this now becomes its own type that the scale bar + timestamp options extend.